README.md

```diff
@@ -43,6 +44,7 @@ If you are using this inside a container, a POSIX-compliant `tar` needs to be in
 * `path` - A list of files, directories, and wildcard patterns to cache and restore. See [`@actions/glob`](https://github.com/actions/toolkit/tree/main/packages/glob) for supported patterns.
 * `key` - An explicit key for restoring and saving the cache
 * `restore-keys` - An ordered list of prefix-matched keys to use for restoring stale cache if no cache hit occurred for key.
+* `enableCrossOsArchive` - An optional boolean when enabled, allows Windows runners to save or restore caches that can be restored or saved respectively on other platforms.
```

Comment:
Nitpicking here but it would good to explicitly mention that default value for this will be false.

Reply:
Good catch